src/main/java/Decide.java

```diff
@@ -545,4 +546,75 @@ void calcFUV() {
 		}
 	}
 
+    /**
+     * Launch Interceptor Condition 6 returns
+     *
+     * @return true if there exists at least one set of nPts consecutive data points such that
+     * least one of the points lies a distance greater than DIST from the line between the first of these nPts points and the last.
+     * If the first and last points of these nPts are identical, then we compare the distance from that point to all
+     * The condition is not met when numPoints < 3.
+     */
+    boolean lic6() {
+
+        if (points.size() < parameters.nPts || points.size() < 3) {
+            return false;
+        }
+
+        for (int i = 0; i <= (points.size() - parameters.nPts); i++) {
+            //if the first and last points are identical, check the distance from the point to the rest of
+            //the points in the
+            if ((points.get(i).getX() == points.get(parameters.nPts - 1).getX()) && (points.get(i).getY() == points.get(parameters.nPts - 1).getY()) && i != parameters.nPts - 1) {
+
+                double sumDist = 0;
+                for (int j = 0; j < parameters.nPts - 2; j++) {
+                    double dist = dist(points.get(0), points.get(j + 1));
+                    sumDist += dist;
```

Comment:
Instead of sum, return true if at least one of the nPts is farther away from the coinciding point than dist